Dockerfile

```diff
@@ -1,17 +1,21 @@
 FROM metalstack/builder:latest as builder
 
+FROM r.metal-stack.io/metal/supermicro:2.5.2 as sum
+
 FROM letsdeal/redoc-cli:latest as docbuilder
 COPY --from=builder /work/spec/metal-core.json /spec/metal-core.json
 RUN redoc-cli bundle -o /generate/index.html /spec/metal-core.json
 
-FROM alpine:3.13
+FROM ubuntu:20.04
```

Comment:
What about still using alpine and installing `gcompat`? I think this would enable running `sum`:

```
❯ docker run --rm -it alpine                                                                                                                                                                                                                                                                                                                                                                                                                                                             
# copying sum into container from the outside..
/ # /sum                                                                                                                                                                                                                                          
/bin/sh: /sum: not found                                                                                                                                                                                                                          
/ # apk add gcompat                                                                                                                                                                                                                               
fetch https://dl-cdn.alpinelinux.org/alpine/v3.14/main/x86_64/APKINDEX.tar.gz                                                                                                                                                                     
fetch https://dl-cdn.alpinelinux.org/alpine/v3.14/community/x86_64/APKINDEX.tar.gz                                                                                                                                                                
(1/3) Installing musl-obstack (1.2.2-r0)                                                                                                                                                                                                          
(2/3) Installing libucontext (1.1-r0)                                                                                                                                                                                                             
(3/3) Installing gcompat (1.0.0-r2)                                                                                                                                                                                                               
OK: 6 MiB in 17 packages                                                                                                                                                                                                                          
/ # /sum                                                                                                                                                                                                                                          
Supermicro Update Manager (for UEFI BIOS) 2.5.2 (2021/01/12) (x86_64)                                                                                                                                                                             
Copyright(C) 2013-2021 Super Micro Computer, Inc. All rights reserved.
...
```

Reply:
i take care of it